bin/get_data.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+const fs = require('fs');
+const TOML = require('@iarna/toml');
+
+const header = 'function addEventListener() {}; ';
```

Comment:
I forgot to ask this earlier, but what's `function addEventListener() {};` for here?

Reply:
`redirector.js` calls the function `addEventListener` which is defined somewhere in Cloudflare's implementation. To parse the source with `eval` without an error, a stub `addEventListener` is defined:
 
```
var workerPaths = eval(header + body + footer);
```

Reply:
Suggest adding a comment with a clue as to why here?

Reply:
Also, just to add: I know we're jumping through quite a few hoops here just to keep the worker routes defined in the JS. Technically, they don't _have_ to be. There's nothing in the worker itself that requires tham.

Because we'll likely update routes pretty infrequently, I'd personally be OK with having them defined in the `.toml` file (seems like less logic and code wrangling). But I understand if we think it'll be clearer overall to have them defined in the same file.